aten/src/ATen/native/BlasKernel.cpp

```diff
@@ -283,6 +267,18 @@ bool gemv_use_fast_path<at::BFloat16>(
       beta == 0.0;
 }
 
+static void bf16_gemv_trans(
+    const int m,
+    const int n,
+    const at::BFloat16 alpha,
+    const at::BFloat16* a,
+    const int lda,
+    const at::BFloat16* x,
+    const int incx,
+    const at::BFloat16 beta,
+    at::BFloat16* y,
+    const int incy);
+
 void bf16_gemv_trans(
```

Comment:
Static functions do not need forward declarations, do they?
```suggestion
static void bf16_gemv_trans(
```